svc-api/handle/handlers.go

```diff
@@ -152,7 +152,7 @@ func getService(microServices []string, uuid string) models.ServiceRoot {
 					serviceRoot.UpdateService = &models.Service{OdataID: servicePath}
 				}
 			}
-		case "TelemetryService":
+		case "Telemetry":
```

Comment:
Telemetry is a redfish service. I think we should rename it as TelemetryService.

Reply:
Yes, i have edited it back to TelemetryService